### DISCUSSION TITLE
Crear releases y un CHANGELOG

### DISCUSSION THREAD

**Discussion by RodrigoTomeES** [elpatronaco/pccomponentes-buy-bot#19] [Ideas]
Hola buenas,

Creo que estaría bien crear releases de commits en los que el bot este probado así la gente podría usar versiones estables del bot y ver que tiene de nuevo en una versión respecto a otra,

 Un saludo,

Rodrigo

**Comment by elpatronaco** [comment]
Se podría hacer, no tengo ni idea cómo porque nunca lo he hecho xd

**Reply by elpatronaco** [reply]
Voy a probar de crear la "primera" versión

**Reply by elpatronaco** [reply]
qué le pongo, 1.0 no? Y a partir de ahí sería subir décimas para cambios grandes y centésimas para arreglos de bugs o algo

**Reply by RodrigoTomeES** [reply]
Es muy facil te vas a la sección de releases y ahí clicas en _draft a new release_ y dentro seleccionas el commit a la que hacer referencia la release y le pones una tag, puedes ver aquí como se versiona http://semver.org/. Básicamente es asi:

- MAJOR version when you make incompatible API changes,
- MINOR version when you add functionality in a backwards compatible manner, and
- PATCH version when you make backwards compatible bug fixes.

Y luego pues añadir un título a la release y una descripción si quieres. Y deberías añadir el zip a descargar con el repo (creo que no lo añade solo).

**Reply by elpatronaco** [reply]
Acabo de hacer una release siguiendo repos así famosetes, míratela a ver si hay que cambiar algo

**Reply by RodrigoTomeES** [reply]
Lo veo genial 👌

**Reply by elpatronaco** [reply]
Sé que en el tag abría que poner MAJOR/MINOR/PATCH pero creo q la versión da más información, así puedes buscar por número para ver q cambios hay sabes

**Reply by RodrigoTomeES** [reply]
No no, osea esta bien como lo has puesto, creo que no me he explicado bien.

- Major hace referencia a versiones v.x.0.0
- Minor hace referencia a versiones v.1.x.0
- Patch hacer referencia a versiones v.1.0.x

Pero se pone con numero, el nombre de major, minor y patch es como se conoce a una actualización de ese tipo.

**Reply by elpatronaco** [reply]
Aahh hahaha ya te entiendo
